adhocteam/Head-Start-TTADP
# Rolling deploys

**Description of change**

* use multiple manifest.yml files
* Allows rolling deploys on main app
* Makes the maintenance app much faster to deploy

**How to test**

Sandbox deploy should work https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP/2044/workflows/b7ad5217-3197-4d50-a291-8f8dde64ffff

**Issue(s)**
* https://ocio-jira.acf.hhs.gov/browse/TTASH-58

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
